docs/sources/reference/loki-http-api.md

```diff
@@ -24,7 +24,7 @@ Authorization needs to be done separately, for example, using an open-source loa
 These endpoints are exposed by the `distributor`, `write`, and `all` components:
 
 - [`POST /loki/api/v1/push`](#ingest-logs)
-- [`POST /otlp/v1/logs`](#ingest-logs-using-otlp)
+- [`POST /otlp`](#ingest-logs-using-otlp)
```

Comment:
This should still be `/otlp/v1/logs` as if you where to call the endpoint without `/v1/logs` I belive it would fail.  Its only the otel collector which adds the `v1/logs` automatically